### PR TITLE
Avoid converting bytes to UTF-8 strings to print, just pass bytes to stdout/err

### PR DESCRIPTION
**Pull request by spastorino**
r? @nikomatsakis

### CODE REVIEW

**Comment by nikomatsakis** [comment]
I don't know this part of the code well, but makes sense to me -- 

r? @alexcrichton for reassignment

**Comment by spastorino** [comment]
@alexcrichton doh, force pushed with the thing fixed

File: src/libstd/process.rs [hunk 1]
```diff
@@ -764,14 +764,15 @@ impl Command {
...
+    /// use std::io::{self, Write};
     /// let output = Command::new("/bin/cat")
     ///                      .arg("file.txt")
     ///                      .output()
     ///                      .expect("failed to execute process");
     ///
     /// println!("status: {}", output.status);
-    /// println!("stdout: {}", String::from_utf8_lossy(&output.stdout));
-    /// println!("stderr: {}", String::from_utf8_lossy(&output.stderr));
+    /// io::stdout().write(&output.stdout);
```
Comment (ollie27) [inline src/libstd/process.rs]: These should use `write_all`, `write` isn't guaranteed to write all of the bytes. Also, as the travis error says, you need to handle the return value with an `.unwrap()` or something.

File: src/libstd/process.rs [hunk 2]
```diff
@@ -951,14 +952,16 @@ impl Stdio {
...
     /// use std::process::{Command, Stdio};
+    /// use std::io::{self, Write};
     ///
     /// let output = Command::new("rev")
     ///     .stdin(Stdio::inherit())
     ///     .stdout(Stdio::piped())
     ///     .output()
     ///     .expect("Failed to execute command");
     ///
-    /// println!("You piped in the reverse of: {}", String::from_utf8_lossy(&output.stdout));
```
Comment (tamird) [inline src/libstd/process.rs]: This seems wrong, doesn't it? This was previously an atomic write, and now is not?

**Pull request merged**